frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -25,25 +25,34 @@ import {
 } from "../api/generated";
 import { getOrThrowError } from "../helpers/util";
 
-export const womanLocalized = {
-  defaultMessage: "I identify as a woman",
-  description: "Message for woman",
-};
-
-export const indigenousLocalized = {
-  defaultMessage: "I am indigenous",
-  description: "Message for indigenous",
-};
-
-export const minorityLocalized = {
-  defaultMessage: "I identify as a member of a visible minority group",
-  description: " Message for minority group",
-};
+export const employmentEquityStatements = defineMessages({
```

Comment:
Did you want to add a set of strings for the search page that use the same keys?  It looks like these don't quite match Jerbo's option strings either.  Could be out of scope, I guess.

![image](https://user-images.githubusercontent.com/8978655/180833243-42b0bc4e-4426-412c-9605-2f600d65924a.png)

Reply:
Good catch. I think this is out of scope though and have created a new issue:

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/3484